crypt/crypt.go

```diff
@@ -87,7 +87,7 @@ func NewFs(name, rpath string) (fs.Fs, error) {
 	remotePath := path.Join(remote, cipher.EncryptFileName(rpath))
 	wrappedFs, err := fs.NewFs(remotePath)
 	// if that didn't produce a file, look for a directory
-	if err != fs.ErrorIsFile {
+	if err != fs.ErrorIsFile && err != nil {
```

Comment:
This is what broke the crypt unit tests!

Reply:
no longer needed so removed